### PR TITLE
[Redesign] Fix visual issues on narrow screens

### PR DESCRIPTION
**Pull request by loic-sharma**
Fixes a few problems. Addresses https://github.com/NuGet/NuGetGallery/issues/8719

I recommend reviewing without whitespace changes:

image

### 1. Prevent README images from overflow.

<details>
<summary>Before...</summary>

image

</details>

<details>
<summary>After...</summary>

image

</details>

### 2. Fix Bootstrap grid to add gutter and avoid an unnecessary scroll bar

<details>
<summary>Before...</summary>

Notice the scrollbar and the lack of gutter.

image

</details>

<details>
<summary>After...</summary>

image

</details>

### 3. Improve the installation instructions scrollbar

<details>
<summary>Before...</summary>

Notice the installation instructions do not stretch all the way across.

Notice there is space below the scrollbar.

image

</details>

<details>
<summary>After...</summary>

image

</details>

### CODE REVIEW

File: src/Bootstrap/less/theme/page-display-package-v2.less [hunk 1]
```diff
@@ -506,5 +506,12 @@
         margin-top: 0px;
       }
     }
+
+    #readme-tab {
+      // Don't allow README images to overflow
+      img {
+        max-width: 100%;
+      }
+    }
```
Comment (loic-sharma) [inline src/Bootstrap/less/theme/page-display-package-v2.less:515]: Previously image overflow was prevented by this:

https://github.com/NuGet/NuGetGallery/blob/77c5f6e57e0e53e8ef52b3e433571b2469681e25/src/Bootstrap/less/theme/common-readme.less#L14-L32

This code applies to collapsible README views, which is used on upload and manage package pages.

File: src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml [hunk 2]
```diff
@@ -215,791 +215,808 @@
 @helper MakeLicenseSpan(CompositeLicenseExpressionSegment segment) {<span>@segment.Value</span>}
 
 <section role="main" class="container main-container page-package-details-v2">
-    <div class="row">
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml:218]: We currently have two Bootstrap rows nested within each other without any columns in the middle:

image

This caused issues on smaller screens:

image

Notice the scrollbar and the lack of gutter. After this fix:

image

File: src/Bootstrap/less/theme/page-display-package-v2.less [hunk 3]
```diff
@@ -414,55 +420,6 @@
       font-weight: 600;
       text-decoration: underline;
     }
-
-    .tab-content {
```
Comment (loic-sharma) [inline src/Bootstrap/less/theme/page-display-package-v2.less:418]: This is V1 installation instruction styling which isn't used in the redesign.

**Review by dannyjdev — APPROVED** [review]
Looks good! :shipit:

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**